OCHuds/Classes/SpinnerHUD.swift

```diff
@@ -26,13 +26,25 @@ internal var topMostViewController: UIViewController? {
 
 // TODO: Change to OCExtensions
 internal extension UIView {
-    func centerSuperview(withSize size: CGSize) {
+    func centerSuperview(withSize size: CGSize, constantX: CGFloat = 0, constantY: CGFloat = 0) {
```

Comment:
Maybe, but I trying to follow the constraints attribute names, margin and padding sound so Android lol